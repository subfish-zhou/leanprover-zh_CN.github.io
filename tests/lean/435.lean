variable {α} (op : α → α → α)
local infixr:60 " ∙ " => op
example (x y : α) : x ∙ y = y ∙ x := sorry

infixr:60 " *** " => op -- Error, only local notation can reference section variables
