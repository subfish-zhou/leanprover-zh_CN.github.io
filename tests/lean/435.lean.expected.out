435.lean:3:37-3:42: warning: declaration uses 'sorry'
435.lean:5:21-5:23: error: unknown identifier 'op' at quotation precheck; you can use `set_option quotPrecheck false` to disable this check.
